### ISSUE TITLE
HTML generated by markdown truncated after routing

### ISSUE THREAD

**Issue by minno726** [David-OConnor/seed#117]
When I generate markdown from HTML and try to display it, only the first element appears. This only appears to happen after routing, for some reason. Here's a small reproduction, based on the quickstart repo:

```rust
#[macro_use]
extern crate seed;
use seed::prelude::*;

// Model

struct Model {
    pub state: i32,
}

impl Default for Model {
    fn default() -> Self {
        Self { state: 0 }
    }
}

// Update

#[derive(Clone, Debug)]
enum Msg {
    SetState(i32),
}

fn update(msg: Msg, model: &mut Model, _: &mut Orders<Msg>) {
    match msg {
        Msg::SetState(state) => { model.state = state; },
    }
}

// View

fn view(model: &Model) -> Vec<El<Msg>> {
    match model.state {
        0 => vec![a![attrs![At::Href => "/test"], "Go to test page"]],
        1 => {
            let body = El::from_markdown("# Test1\n\n## Test2");
            log!(body);
            body
        },
        _ => unimplemented!(),
    }
}

fn route(url: &seed::Url) -> Msg {
    if url.path.is_empty() {
        return Msg::SetState(0);
    }

    match &*url.path[0] {
        "test" => Msg::SetState(1),
        _ => Msg::SetState(0),
    }
}

#[wasm_bindgen]
pub fn render() {
    seed::App::build(Model::default(), update, view)
        .routes(route)
        .finish()
        .run();
}
```

After clicking on the link to go to /test, the result only has the "Test1" line. The exact html is `<h1 xmlns="http://www.w3.org/1999/xhtml">Test1</h1>`. However, if you remove all of the routing by changing `view` to

```rust 
fn view(model: &Model) -> Vec<El<Msg>> {
    let body = El::from_markdown("# Test1\n\n## Test2");
    log!(body);
    body
}
```
then both lines of the markdown appear.

**Comment by David-OConnor** [comment]
I don't have the fix yet, but it appears this has something to do with state, rather than routing, and appears only when returning a `Vec<El>`. Ie if we use event logic instead of routing to switch state, the bug still occurs. But, if we have the view fn return an `El`, ie by wrapping the `Vec<El>` produced by `from_markdown`, it works:

```rust
fn view(model: &Model) -> El<Msg> {
    match model.state {
        0 => a![simple_ev(Ev::Click, Msg::SetState(1)), "Go to test page"],
        1 => {
            let body = El::from_markdown("# Test1\n\n## Test2");
            log!(body);
            div![body]
        },
        _ => unimplemented!(),
    }
}
```

**Comment by David-OConnor** [comment]
Fixed in latest commit and v0.3.5 release.

**Comment by minno726** [comment]
I can confirm, the new version fixes the problem I was having.
